src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -844,7 +844,7 @@ def get_events(
             raise ValueError("first and last cannot be used together")
 
         account_event_table = cls.get_account_event_table()
-        if not account_event_table:
+        if account_event_table == "NONE":
```

Comment:
is this a BCR change? if not, can we keep checking for the empty case too?

Reply:
I'm not sure but you're probably right that we shouldn't make that assumption, so I'll change it back and just add the check for the "NONE" case